WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackSetupWebViewController.swift

```diff
@@ -130,8 +145,58 @@ extension JetpackSetupWebViewController: WKNavigationDelegate {
 private extension JetpackSetupWebViewController {
     enum Constants {
         static let jetpackInstallString = "https://wordpress.com/jetpack/connect?url=%@&mobile_redirect=%@&from=mobile"
-        // TODO: update this URL with woocommerce:// when https://github.com/Automattic/wp-calypso/pull/65715 is merged.
-        static let mobileRedirectURL = "wordpress://jetpack-connection"
+        static let mobileRedirectURL = "woocommerce://jetpack-connected"
+    }
+
+    enum JetpackSetupWebSteps: CaseIterable {
```

Comment:
super nit: we usually name enum in the singular form? (Constants is an exception but it doesn't have any cases)
```suggestion
    enum JetpackSetupWebStep: CaseIterable {
```

Reply:
Also handled in e95804e94f3beec28dd631d2947df4ea545c0672 👍